pkg/volume/util.go

```diff
@@ -29,30 +29,63 @@ import (
 	"k8s.io/kubernetes/pkg/watch"
 
 	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/api/resource"
 )
 
-// ScrubPodVolumeAndWatchUntilCompletion is intended for use with volume Recyclers.  This function will
+func GetAccessModesAsString(modes []api.PersistentVolumeAccessMode) string {
```

Comment:
I will refactor this out.   The func in api/helpers.go is mine, too, and originally came from this PR but got merged in a different PR.